src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -1,26 +1,24 @@
-package racingcar.application
+package racingcar.domain
 
-class Car private constructor(
+class Car(
     val name: String,
-    var position: Int,
+    position: Int = STARTING_POSITION,
 ) {
     init {
         require(name.trim().length <= CAR_NAME_LENGTH_LIMIT) { "자동차 이름은 5자를 초과할 수 없습니다" }
```

Comment:
이렇게 바꿔두면 길이제한 상수만 변경하면 예외메시지도 같이 변경이 될 것 같네요. 😃
```suggestion
        require(name.trim().length <= CAR_NAME_LENGTH_LIMIT) { "자동차 이름은 ${CAR_NAME_LENGTH_LIMIT}자를 초과할 수 없습니다" }
```